localstack.yaml

```diff
@@ -0,0 +1,177 @@
+      repository: https://github.com/localstack/localstack
+      tag: v${{package.version}}
+      expected-commit: aa795ed1c04a8285c01c96aa7ab79c17489f5766
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/share/localstack
+      mkdir -p ${{targets.destdir}}/tmp/localstack
+
+  - name: Build
+    runs: |
+      make install-runtime
+      make entrypoints
+      python${{vars.py-version}} -m build
+
+  - runs: |
+      # We use uv here as pip backtracks in resolution
+      uv venv -p python${{vars.py-version}} --seed
+      # Install the wheel from file with the runtime extras
+      uv pip install ./dist/localstack_core-${{package.version}}-py3-none-any.whl[runtime]
+      uv pip install awscli-local
+
+      # copy the entry_points definition into .venv
+      install -m644 ./localstack-core/localstack_core.egg-info/entry_points.txt \
+        .venv/lib/python${{vars.py-version}}/site-packages/localstack_core-${{package.version}}.dist-info/entry_points.txt
+      echo -n "/usr/share/localstack/.venv/lib/python${{vars.py-version}}/site-packages/localstack_core-${{package.version}}.dist-info/entry_points.txt" \
+        >.venv/lib/python${{vars.py-version}}/site-packages/localstack_core-${{package.version}}.dist-info/entry_points_editable.txt
+
+      mv .venv ${{targets.destdir}}/usr/share/localstack/
+
+  - name: Install lambda-runtime
+    runs: |
+      ${{targets.destdir}}/usr/share/localstack/.venv/bin/python3 -m localstack.cli.lpm install lambda-runtime
+
+  - runs: |
+      # Fix venv paths
+      find ${{targets.destdir}}/usr/share/${{package.name}}/.venv/bin/ -type f | xargs sed -i "s|/home/build|/usr/share/${{package.name}}|g"
+      sed -i "s|include-system-site-packages = false|include-system-site-packages = true|g" ${{targets.destdir}}/usr/share/${{package.name}}/.venv/pyvenv.cfg
+
+  - runs: |
+      # force reinstall to address cve in previous versions
+      ${{targets.destdir}}/usr/share/localstack/.venv/bin/python3 -m pip install --upgrade --force-reinstall amazon-kclpy cryptography
+
+  - name: Cleanup
+    runs: |
+      find ${{targets.destdir}}/usr/share/${{package.name}}/.venv -name "*.pyc" -delete
+      find ${{targets.destdir}}/usr/share/${{package.name}}/.venv -name "__pycache__" -exec rm -rf {} +
+      find ${{targets.destdir}}/usr/share/${{package.name}}/.venv/lib/python${{vars.py-version}}/site-packages/ -type d -name "tests" -exec rm -rf {} +
+
+  - name: Install scripts
+    working-directory: bin
+    runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/bin
+      for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+        install -Dm755 $f ${{targets.contextdir}}/usr/bin/$f
+      done
+
+  - name: Mark community version
+    runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/lib/localstack
+      echo "${{package.version}} (Built by Chainguard)" > ${{targets.contextdir}}/usr/lib/localstack/.community-version
+
+subpackages:
+  - name: "${{package.name}}-compat"
```

Comment:
Can we have stat based or some basic test of these subpackages to make sure they are exactly there where they are expected